lib/v2/juejin/events.js

```diff
@@ -0,0 +1,44 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const city = ctx.params.city === 'all' ? '' : ctx.params.city;
+    let cityName = '';
+
+    const response = await got({
+        method: 'post',
+        url: 'https://api.juejin.cn/event_api/v1/event/event_list',
+        json: {
+            city: city === 'all' ? '' : city,
+            count: 20,
```

Comment:
```suggestion
            count: ctx.query.limit ? parseInt(ctx.query.limit) : 20,
```
This way users can specify the number of items they want using common parameters